main.go

```diff
@@ -3,65 +3,78 @@ package main
 import (
 	"fmt"
 	"os"
-
-	"github.com/aws/aws-sdk-go/aws/ec2metadata"
-	"github.com/aws/aws-sdk-go/aws/session"
-	"github.com/aws/aws-sdk-go/service/elb"
 )
 
-const StateInService = "InService"
+func usage() {
+	fmt.Printf("USAGE: %s <check|register|deregister> <load-balancer-name>\n", os.Args[0])
+	os.Exit(2)
+}
+
+func must(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
+
+func mustPretty(err error) {
+	if err != nil {
+		fmt.Printf("ERROR: %s\n", err)
+		os.Exit(1)
+	}
+}
 
 func main() {
-	var err error
+	if len(os.Args) != 3 {
+		usage()
+	}
 
-	if len(os.Args) != 2 {
-		fmt.Printf("USAGE: %s <load-balancer-name>", os.Args[0])
-		os.Exit(2)
+	command := os.Args[1]
```

Comment:
We already have the template, why do we invent something new here?

Even if it might be a bit bigger then needed.